### PR TITLE
Sous Chefs Adoption

### PR DESCRIPTION
**Pull request by ramereth**
# Description

Describe what this change achieves

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -18,4 +18,4 @@
 supports 'ubuntu'
 
 chef_version '>= 15.0'
-gem 'docker-api', '~> 1.34.0'
```
Comment (tas50) [inline metadata.rb:21]: This should be >= 1.34, < 3 probably so it works with older versions of chef correctly

File: Berksfile [hunk 2]
```diff
@@ -2,6 +2,9 @@ source 'https://supermarket.chef.io'
 
 metadata
 
+# XXX: Temporary for testing
+cookbook 'etcd', github: 'sous-chefs/etcd', branch: 'sous-chefs-adoption'
```
Comment (ramereth) [inline Berksfile:6]: This will get removed before merging.

**Review by tas50 — DISMISSED** [review]

File: libraries/docker_service_manager_systemd.rb [hunk 3]
```diff
@@ -26,6 +26,16 @@ class DockerServiceManagerSystemd < DockerServiceBase
         not_if { docker_name == 'default' && ::File.exist?('/lib/systemd/system/docker.socket') }
       end
 
+      template '/etc/systemd/system/containerd.service' do
```
Comment (tas50) [inline libraries/docker_service_manager_systemd.rb:29]: I don't think containerd goes all the way back to 17.12. You might want to figure out when it came out and then mark that as the oldest version of Docker supported.

Comment (ramereth) [reply libraries/docker_service_manager_systemd.rb:29]: That thought had occurred to me. I'll look into that.

**Comment by tas50** [comment]
Very nice cleanup. Thanks @ramereth

**Comment by ramereth** [comment]
@tas50 still OK with a minor bump on this?

**Pull request merged**